docs/src/components/ReactExample/index.tsx

```diff
@@ -69,6 +76,7 @@ const ReactExample = ({ exampleId, background = "white", minHeight, maxHeight }:
 
   return (
     <Example
+      responsive={responsive}
```

Comment:
do not forget also to pass responsive prop in `docs/src/template/Sandbox` ;)

Reply:
It shouldn't be necessary because on full app viewport ruler should always be there, and in `Example` `responsive` is `true` by default.